### ISSUE TITLE
Improve Animation Player Usability

### ISSUE THREAD

**Issue by ExquisiterEmil** [godotengine/godot#72088]
### Godot version

4.0beta15

### System information

Windows10, AMD Ryzen 9 3900XT, RTX3070, Vulkan

### Issue description

Currently, there are several issues, that constantly throw me and my colleagues off when using the Animation Player. Some of them are UI-related others are about keyframe functionality. I am consciously not posting this under "proposals", because I think these are serious issues, that make the AnimationPlayer an unpleasant experience.

### Issue 1: Zoom tool

The Zoom tool (or CTRL+Mouse wheel) animation player should definitely zoom in on the current position, not the beginning of the track as it does right now! The current behavior is bad UX, any other timeline-based editing tool (animations, video, audio, etc.) zooms in on the cursor position.
![AnimationPlayerZooming](https://user-images.githubusercontent.com/53947784/214716432-f7cbd59e-de5b-472b-b07e-32dbb09b3820.gif)

### Issue 2: Missing Copy + Paste of keyframes

Two questions here: why can we CTRL+D (duplicate) keyframes, but not CTRL+C and CTRL+V (copy+paste)? And since I can't do it, why does it instead copy the AnimationPlayer NODE itself and paste it, even though it isn't even selected in the inspector? 

![KeyframePasting](https://user-images.githubusercontent.com/53947784/214717398-1f127fda-e0cd-4218-b8ab-65843dc6db48.gif)

There should be an option to copy any keyframe and paste it on the current playback position. Also, I think it should be possible to copy a keyframe on another track of the same type.

### Issue 3: Missing duplication of entire Tracks

It would be nice to have the option to duplicate an entire animation track, just with a different path that it modifies. 

### Issue 4: Having to make both Animation AND Library unique, when duplicating

If I have an AnimationPlayer, I want to duplicate it and edit the animations of the second player only, I have to not only make the animation unique, but also the animation library? There is at least a "Make unique (recursive)" option missing on the library, but maybe the library should make its Animations unique altogether by default? I haven't used Animation Libraries that much, so correct me if that is bad.

### Issue 5: Bugs and issues with AudioPlaybackTracks, partly tracked in #71810 

The AudioPlaybackTrack is both blessing and curse, it's very nice, but many things need to be ironed out, such as those in the mentioned issue. Also, I think audio should not be played when positioning the cursor somewhere, but only upon playback. E.g. why would I want to hear the death-scream of a character each time I reposition its ragdoll? It's mostly undesirable and annoying and if you have long audio, it can be unbearable. 

### Issue 6: Can't drag (audio) keyframes across tracks
This would be nice to have, e.g. you want two have the same sound on two Playback tracks, or you accidentally dragged an audio file from the file system onto the wrong track. This could also work for other keyframe types, and might even go hand-in-hand with Issue 2. The reason this is important for audio files specifically is that users of any audio editing program (think Audacity/Reaper/...) would intuitively try to do that, especially since you can also change the playback range.

![image](https://user-images.githubusercontent.com/53947784/214719769-070fbda7-87a2-4c77-955d-9188b51f1374.png)

You can see, that there is a lot that could be improved with the current Animation Player. It is such an important feature for an Engine, yet so many usability features one would expect are just not there. I believe these improvements would be both fairly minor to implement, but at the same time benefit a majority of the users greatly.

### Steps to reproduce

Try to use the AnimationPlayer with several tracks and track types and you will encounter the issues described above. Above descriptions should suffice.

### Minimal reproduction project

Here is an Animation Playback Track with some audio, where you can try a bunch of that stuff if that's of any help.
[AnimationPlayerUsability.zip](https://github.com/godotengine/godot/files/10504738/AnimationPlayerUsability.zip)

**Comment by Calinou** [comment]
Please open one proposal per suggested feature to make the discussion on each feature easier to follow.

**Comment by ExquisiterEmil** [comment]
Thanks for the comment, when I find some time the next days, I will create a proposal for each and every one (except issue 5) and update this post with links. 
Part of why I made this post, is because I kept running into these "issues" and wanted to have a central location to discuss pain points with the AnimationPlayer. I understand, that tracking implementation progress is easier with dedicated issue posts, but I thought such a post might help to assess the state of usability of the tool in general.
